components/event/EventTabs.js

```diff
@@ -0,0 +1,63 @@
+export function EventTabs({ tabs, eventType, setEventType }) {
+  const classNames = (...classes) => classes.filter(Boolean).join(" ");
+  const changeTab = (e, value) => {
+    setEventType(value);
+    if (!value) {
+      setEventType(tabs.find((tab) => tab.title === e.target.value).key);
+    }
+  };
+
+  return (
+    <div>
+      <div className="sm:hidden">
+        <label htmlFor="tabs" className="sr-only">
+          Select a tab
+        </label>
+        <select
+          id="tabs"
+          name="tabs"
+          onChange={(e) => changeTab(e)}
+          className="block w-full rounded-md border-gray-300 focus:border-indigo-500 focus:ring-indigo-500"
+          defaultValue={tabs.find((tab) => tab.key === eventType)?.title}
+        >
+          {tabs.map((tab) => (
+            <option key={tab.key}>{tab.title}</option>
+          ))}
+        </select>
+      </div>
+      <div className="hidden sm:block">
+        <div className="border-b border-gray-200">
+          <nav className="-mb-px flex" aria-label="Tabs">
+            {tabs.map((tab) => (
+              <a
+                key={tab.key}
+                href={tab.href}
+                onClick={(e) => changeTab(e, tab.key)}
+                className={classNames(
+                  tab.key === eventType
+                    ? "border-indigo-500 text-indigo-600"
+                    : "border-transparent text-gray-500 hover:text-gray-700 hover:border-gray-300",
+                  "w-1/4 py-4 px-1 text-center border-b-2 font-medium text-sm flex justify-center items-center gap-2 cursor-pointer"
+                )}
+                aria-current={tab.key === eventType ? "page" : undefined}
+              >
+                {
+                  <tab.icon
+                    className={`${
+                      tab.key === "all" || tab.key === "cfpOpen"
```

Comment:
Why the different sizes for the `all` and `cfpopen` types?

Reply:
They are a little bigger compared to the other two, which looks odd

This is all icons having `text-xl`
![image](https://user-images.githubusercontent.com/95094057/212310639-8bb00941-dca0-4665-a531-337e8b8fbc26.png)

and This is two of the first icons having `text-lg` and the other two `text-xl`
![image](https://user-images.githubusercontent.com/95094057/212310802-b82284a5-f44b-4797-a19a-1902634e142a.png)

Reply:
Alright seems fair